### ISSUE TITLE
Unknown error with Turnip Prophet

### ISSUE THREAD

**Issue by NikNovak99** [mikebryant/ac-nh-turnip-prices#430]
**Things to check**
- Have you forced a refresh in your browser (e.g. ctrl+F5 in Chrome/Firefox)?
  - yes
- Have you made sure the first time buyer option is correct? If you're unsure, try it both ways
  - yes
- Have you time-travelled this week? Travelling backwards resets the prices, and therefore you must not put prices in from both before & after the time-travel
  - no
![Screenshot 2020-12-05 at 4 58 43 PM](https://user-images.githubusercontent.com/75548594/101267126-2fc4bf80-371b-11eb-95b3-641012d99c0e.png)

**Describe the bug**
When I put in all the information, there is still an error and I can't find out my pattern. It happened last week as well. Same problem as now.

**Permalink to your prices**
https://turnipprophet.io?prices=103.93.89.85.81.76.113.192.265.158.137.85.77&first=true&pattern=3

**Desktop (please complete the following information):**
 - Browser [Chrome]
 - Version [Unknown]

**Comment by peter50216** [comment]
Have you made sure the first time buyer option is correct?

From your description: `It happened last week as well.`, so the first time buyer option should be set to "no", and there will be prediction.

**Comment by NikNovak99** [comment]
Yes, all the information is correct. Including the first buyer information.
